EIPS/eip-template.md

```diff
@@ -0,0 +1,235 @@
+function removeListDelegate(address owner, address delegate, bytes32 list) public;
+```
+
+#### removeListDelegateSigned ([see Meta transactions](#Meta-transactions))
+**OPTIONAL** implements a function to remove a delegate from an owner's revocation list in a namespace with a raw signature.
+```solidity
+function removeListDelegateSigned(address owner, address delegate, bytes32 list, uint8 sigV, bytes32 sigR, bytes32 sigS) public;
+```
+
+### Events
+
+**RevocationStatusChanged**
+**MUST** be emitted when `changeStatus`, `changeStatusSigned`, `changeStatusDelegate`, or `changeStatusDelegateSigned` was successfully executed.
+
+```solidity
+event RevocationStatusChanged(
+    address indexed namespace,
+    bytes32 indexed list,
+    bytes32 indexed key,
+    bool revoked
+);
+```
+
+**RevocationStatusesChanged**
+**MUST** be emitted when `batchChangeStatuses`, `batchChangeStatusesSigned`, `batchChangeListStatuses`, or `batchChangeListStatusesSigned` was successfully executed.
+
+```solidity
+event RevocationStatusesChanged(
+    address[] indexed namespaces,
+    bytes32[] indexed lists,
+    bytes32[] indexed keys,
+    bool[] revoked
+);
+```
+
+**ListOwnerChanged**
+**MUST** be emitted when `changeListOwner` or `changeListOwnerSigned` was successfully executed.
+
+```solidity
+event ListOwnerChanged(
+    address indexed namespace,
+    address indexed newOwner,
+    bytes32 indexed list
+);
+```
+
+**DelegateAdded**
+**MUST** be emitted when `addListDelegate` or `addListDelegateSigned` was successfully executed.
+
+```solidity
+event ListDelegateAdded(
+    address indexed namespace,
+    address indexed delegate,
+    bytes32 indexed list
+);
+```
+
+**DelegateRemoved**
+**MUST** be emitted when `removeListDelegate` or `removeListDelegateSigned` was successfully executed.
+
+```solidity
+event ListDelegateRemoved(
+    address indexed namespace,
+    address indexed delegate,
+    bytes32 indexed list
+);
+```
+
+### Meta transactions
+
+This section uses the following terms: 
+
+**`transaction signer`:** An Ethereum address that signs arbitrary data for the contract to execute **BUT** does not commit the transaction. 
+**`transaction sender`**: An Ethereum address that takes signed data from a **transaction signer** and commits it wrapped with its own signature to the smart contract.
+
+An address (**transaction signer**) **MAY** be able to deliver a signed payload off-band to another address (**transaction sender**) that initiates the Ethereum interaction with the smart contract. The signed payload **MUST** be limited to be used only once ([Signed Hash](#Signed-Hash) + [nonces](#Nonce)).
+
+#### Signed Hash
+
+The signature of the **transaction signer** **MUST** conform [EIP-712](/EIPS/eip-712), in its state that is proposed in [`ethereum/EIPs/issues/5475`](https://github.com/ethereum/EIPs/issues/5475). This helps users understand what the payload they're signing consists of & it improves the protection against replay attacks.
```

Comment:
```suggestion
The signature of the **transaction signer** **MUST** conform [EIP-712](./eip-712.md). This helps users understand what the payload they're signing consists of & it improves the protection against replay attacks.
```
Required: External links (even to this GitHub repository) aren't allowed.
Required: Internal links must be used when applicable.